src/app/common/paging/pageable-table/pageable-table.component.spec.ts

```diff
@@ -31,24 +29,28 @@ import { PipesModule } from '../../pipes.module';
 			{{ index }}
 		</ng-template>
 
-		<ng-template named-template="empty-table">
-			{{ emptyTableContent }}
+		<ng-template named-template="table-no-data">
```

Comment:
@jrassa is this intended to represent an error state? Otherwise, what's the distinction between "table-no-data" and "table-no-results" ?

Reply:
Nevermind.. I saw your note below. Probably worth documenting either inline via comments or in the README that we can/should expand :-)